composer.json

```diff
@@ -13,9 +13,9 @@
     ],
     "homepage": "http://github.com/bshaffer/oauth2-server-httpfoundation-bridge",
     "require":{
-        "php":">=5.3.0",
+        "php":">=7.0.0",
         "bshaffer/oauth2-server-php": ">=0.9",
-        "symfony/http-foundation": ">=2.1"
+        "symfony/http-foundation": ">=5.0"
```

Comment:
This should remain at being compatible with `symfony/http-foundation >= 2.1`, as there's no reason to change that

Reply:
I think we should bump it to >= 3 probably. Running the unit tests with exact version 2.1 generates the following errors:

```
PHPUnit 6.5.14 by Sebastian Bergmann and contributors.

.F................                                                18 / 18 (100%)

Time: 648 ms, Memory: 4.00MB

There was 1 failure:

1) OAuth2\HttpFoundationBridge\ResponseTest::testAddParameters with data set #0 ('[]', array())
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'[]'
+'{}'

/oauth2-server-httpfoundation-bridge/test/OAuth2/HttpFoundationBridge/ResponseTest.php:17```